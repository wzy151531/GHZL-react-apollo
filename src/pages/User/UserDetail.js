import React from 'react';

export const UserDetail = props => {
  return (
    <div>{`${props.match.params.id}'s detail`}</div>
  );
};