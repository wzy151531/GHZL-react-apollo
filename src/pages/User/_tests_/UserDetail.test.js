import React from 'react';
import UserDetail from '../UserDetail';
import renderer from 'react-test-renderer';

test('UserDetail should render correctly', () => {
  const matchProps = {
    params: {
      id: 1,
    },
  };
  const component = renderer.create(
    <UserDetail match={matchProps}/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});