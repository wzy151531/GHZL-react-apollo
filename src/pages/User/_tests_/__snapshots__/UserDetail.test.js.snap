// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDetail should render correctly 1`] = `
<div>
  1's detail
</div>
`;
